ext/json/json.c

```diff
@@ -116,6 +120,7 @@ static PHP_MINIT_FUNCTION(json)
 	/* common options for json_decode and json_encode */
 	PHP_JSON_REGISTER_CONSTANT("JSON_INVALID_UTF8_IGNORE", PHP_JSON_INVALID_UTF8_IGNORE);
 	PHP_JSON_REGISTER_CONSTANT("JSON_INVALID_UTF8_SUBSTITUTE", PHP_JSON_INVALID_UTF8_SUBSTITUTE);
+	PHP_JSON_REGISTER_CONSTANT("JSON_THROW_EXCEPTIONS", PHP_JSON_THROW_EXCEPTIONS);
```

Comment:
I guess you meant deprecate `JSON_PARTIAL_OUTPUT_ON_ERROR`? If so that's really not a good idea IMHO as we would just force some user change their code for no reason. In addition I prefer `JSON_PARTIAL_OUTPUT_ON_ERROR` as it's more clear what it does. I don't see any need to have it the same as it is in PDO.